models/subject.js

```diff
@@ -0,0 +1,27 @@
+const { Schema, model } = require('mongoose')
+const {
+  enums: { LANG_LEVEL_ENUM }
+} = require('~/consts/validation')
+
+const subjectSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  price: {
+    type: Number,
+    required: true
+  },
+  proficiencyLevel: {
```

Comment:
not sure that it can be req, coz math can't have lets say Native language lvl

think about workaround, can we make a generic proflvl that will fit both math and lang?

Reply:
fixed